iroh/src/node/builder.rs

```diff
@@ -483,6 +483,7 @@ where
 
         // Initialize the internal RPC connection.
         let (internal_rpc, controller) = quic_rpc::transport::flume::connection(1);
+        let controller = quic_rpc::transport::boxed::Connection::new(controller);
```

Comment:
So this still boxes the internal RPC connection? You said somewhere that the internal RPC connection circumvents the boxing. If this is the case, maybe add a comment here why/how this works?